src/test/kotlin/no/nav/tilleggsstonader/sak/vedtak/dagligReise/DagligReiseAndelTilkjentYtelseMapperTest.kt

```diff
@@ -0,0 +1,127 @@
+        with(andeler.single()) {
+            assertThat(fom).isEqualTo(fredag)
+            assertThat(tom).isEqualTo(fredag)
+            assertThat(utbetalingsdato).isEqualTo(fredag)
+        }
+    }
+
+    @Test
+    fun `flere reiser med samme fom aggregeres til én andel med summert beløp`() {
+        val mandag = 1 september 2025
+        val beregningsresultat =
+            Beregningsresultat(
+                reiser =
+                    listOf(
+                        lagBeregningsresultatForReise(mandag, beløp = 100),
+                        lagBeregningsresultatForReise(mandag, beløp = 200),
+                    ),
+            )
+        val andeler = beregningsresultat.mapTilAndelTilkjentYtelse(behandlingId)
+        assertThat(andeler).hasSize(1)
+        assertThat(andeler.single().beløp).isEqualTo(300)
+    }
+
+    @Test
+    fun `reiser med ulike fom-datoer gir en andel per dato`() {
+        val mandag = 1 september 2025
+        val tirsdag = 2 september 2025
+        val beregningsresultat =
+            Beregningsresultat(
+                reiser =
+                    listOf(
+                        lagBeregningsresultatForReise(mandag, beløp = 100),
+                        lagBeregningsresultatForReise(tirsdag, beløp = 200),
+                    ),
+            )
+        val andeler = beregningsresultat.mapTilAndelTilkjentYtelse(behandlingId)
+        assertThat(andeler).hasSize(2)
+        with(andeler.first()) {
+            assertThat(fom).isEqualTo(mandag)
+            assertThat(tom).isEqualTo(mandag)
+            assertThat(utbetalingsdato).isEqualTo(mandag)
+        }
+        with(andeler.last()) {
+            assertThat(fom).isEqualTo(tirsdag)
+            assertThat(tom).isEqualTo(tirsdag)
+            assertThat(utbetalingsdato).isEqualTo(tirsdag)
+        }
+    }
+}
+
+private fun lagBeregningsresultatForReise(
+    fom: LocalDate,
+    beløp: Int = 100,
+): BeregningsresultatForReise =
+    BeregningsresultatForReise(
+        perioder =
+            listOf(
+                BeregningsresultatForPeriode(
+                    grunnlag = lagBeregningsgrunnlag(fom),
+                    beløp = beløp,
+                ),
+            ),
+    )
+
+private fun lagBeregningsgrunnlag(fom: LocalDate): Beregningsgrunnlag =
+    Beregningsgrunnlag(
+        fom = fom,
+        tom = fom.plusWeeks(1),
+        prisEnkeltbillett = 50,
+        pris30dagersbillett = 1000,
+        antallReisedagerPerUke = 5,
+        vedtaksperioder =
+            listOf(
+                VedtaksperiodeGrunnlag(
+                    id = UUID.randomUUID(),
+                    fom = fom,
+                    tom = fom,
+                    målgruppe = NEDSATT_ARBEIDSEVNE,
+                    aktivitet = AktivitetType.TILTAK,
+                    antallReisedagerIVedtaksperioden = 5,
+                ),
+            ),
+        antallReisedager = 20,
+    )
```

Comment:
Disse kunne vi gjort pubilc og lagt i en 'offentligTransportTestUtils.kt`? Ser for meg det flere steder vi trenger dette.